src/routes/Raids.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 
-import { Link } from 'react-router-dom'
+import { Link, useRouteMatch } from 'react-router-dom'
 import LoadingSpinner from '#components/loading-spinner'
 import useCollection from '#utils/useCollection'
 
-const AllRaids = () => {
+const Raids = () => {
```

Comment:
Why the change in the naming convention?

Reply:
i borrowed from REST, again, if this is out of line I don't mind changing it back

Reply:
Nothing is really "out of line" I want to continue to perpetuate crowdsourcing and mostly mobocracy.

Reply:
Then this is my proposal 😄 
Open to feedback